<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Listado con Tablewidge de revistas</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_revistas">
    <property name="geometry">
     <rect>
      <x>-5</x>
      <y>110</y>
      <width>941</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>titulo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>precio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>distribidora</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>numero de revista</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha de entrega</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha de devolucion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>borrar</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>editar</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
